content/guides/java/inject-extract.md

```diff
@@ -138,6 +149,6 @@ public void byteBufferInjection() throws Exception {
 public void byteBufferExtraction(ByteBuffer byteBuffer) throws Exception {
   SpanContext spanContext = tracer.extract(Format.Builtin.BINARY, byteBuffer);
   //do something with spanContext...
-
 }
 ```
+The complete code is available [here](https://github.com/instana/instana-java-opentracing/blob/master/src/test/java/com/instana/opentracing/InstanaTracerTest.java).
```

Comment:
I think it's wonderful that you used an example from Instana!

/cc @CodingFabian and @raphw